tests/ReflectionClassTest.php

```diff
@@ -290,4 +290,37 @@ protected function getGettersToCheck()
 
         return $allNameGetters;
     }
+
+    /**
+     * Tests getName() method
+     *
+     * @dataProvider getClassNamesToCheck
```

Comment:
If I'm not mistaken, then `@dataProvider` annotation should go after everything else. Or do as done in other places.

Reply:
Technically it doesn't matter where.  II use style of ReflectionClassTest::testGetModifiers() & ReflectionClassTest::testReflectionMethodParity()